GVRf/Extensions/gradle.properties

```diff
@@ -1,4 +1,9 @@
 android.useDeprecatedNdk=true
 org.gradle.daemon=true
 org.gradle.jvmargs=-Xmx2048M
-LIBS_DIRECTORY=\.\.\/GearVRf-Demos\/gearvrf-libs
\ No newline at end of file
+LIBS_DIRECTORY=\.\.\/GearVRf-Demos\/gearvrf-libs
+
+#to build the platformsdk-support extension uncomment this property and
+#supply the correct path to the platform sdk. Then run
+#./gradlew -Pplatformsdk_support=true platformsdk_support:assembleDebug
+#OVR_PLATFORM_SDK=<path-to-the-oculus-platform-sdk
```

Comment:
onFailure is supposed to be called in case the actual entitlement check fails. If you are in no position to even submit the request you get the exception. It is mentioned in the javadoc.